pkg/envtest/server.go

```diff
@@ -132,3 +147,22 @@ func (te *Environment) Start() (*rest.Config, error) {
 	})
 	return te.Config, err
 }
+
+func getTimeouts() (time.Duration, time.Duration, error) {
+	var startTimeout, stopTimeout time.Duration
+	if envVal := os.Getenv(envStartTimeout); envVal != "" {
+		converted, err := strconv.Atoi(envVal)
```

Comment:
Looks like a good usecase for `parseDuration` https://golang.org/pkg/time/#ParseDuration